lib/fido/views/reset_dialog.dart

```diff
@@ -97,7 +97,15 @@ class _ResetDialogState extends ConsumerState<ResetDialog> {
                           final String errorMessage;
                           // TODO: Make this cleaner than importing desktop specific RpcError.
                           if (e is RpcError) {
-                            errorMessage = e.message;
+                            if (e.status == 'connection-error') {
+                              errorMessage = l10n.l_failed_connecting_to_fido;
+                            } else if (e.status == 'invalid-command') {
```

Comment:
It smells to me that for 'invalid-command' we show that the inserted key is wrong. Is the 'invalid-command' sent from the helper only in this situation and if not, could we have more specific status, for example 'key-mismatch-error', to be used here instead?

Reply:
I pushed a commit that addresses this.